libs/designsystem/modal/src/modal/services/modal.helper.ts

```diff
@@ -169,4 +175,10 @@ export class ModalHelper {
       modal.dismiss();
     });
   }
+
+  public async showAlert(config: AlertConfig): Promise<boolean> {
+    const alert = await this.alertHelper.showAlert(config);
+    const result = await alert.onWillDismiss;
+    return result.data;
+  }
```

Comment:
Isn't this already implemented in `can-dismiss.helper.spec.ts`? 🤔

Reply:
Yes it is :) But it is also needed here to support the old alert implementation, that we are deprecating. I will add that as a comment in the code :)

Reply:
My point being: shouldn't we just import that helper here then and reuse the implementation?